gmsh_utils/gmsh_IO.py

```diff
@@ -662,12 +659,13 @@ def read_gmsh_geo(self, filename: str):
         """
 
         if pathlib.Path(filename).exists():
-            gmsh.initialize()
+            self.reset_gmsh()
```

Comment:
why reset and not initialize?

Reply:
I only want 1 gmsh instance open at the same time, else weird stuff can happen with the geometry ids. Furthermore, if I dont clear the current gmsh data in the gmsh object, the new geo data will be appended, which shouldnt happen